spaceR/spaceR/Controllers/RocketsControlller.swift

```diff
@@ -11,30 +11,28 @@ import UIKit
 
 class RocketsController: UIViewController {
     
-    @IBOutlet weak var tableView: UITableView!
-    
-    @IBOutlet weak var emailIdLabel: UILabel!
-    var emailText = ""
-    private var rockets: [Rocket] = []
+    @IBOutlet weak var rocketTableView: UITableView!
+    private lazy var rocketViewModel = RocketViewModel()
     
     override func viewDidLoad() {
-        super.viewDidLoad()
```

Comment:
I think you accidentally removed `super.viewDidLoad()`, might want to add it back in

Reply:
Ahh thanks bro!